lua/luasnip/loaders/from_vscode.lua

```diff
@@ -187,12 +187,42 @@ function M.load(opts)
 		opts.paths = get_snippets_rtp()
 	end
 
+	opts.paths = vim.tbl_map(expand_path, opts.paths) -- Expand before deduping, fake paths will become nil
```

Comment:
The paths from `get_snippets_rtp` don't need to be expanded anymore, right? In that case, I think this line should be inside the 'if'.

Reply:
I'm not sure if it always brings the absolute path as `rutimepath` may contain paths with `~`, let me check.